setup.py

```diff
@@ -34,11 +34,11 @@
     include_package_data=True,
     python_requires=">=3.8",
     install_requires=[
-        "tutor>=17.0.0,<18.0.0",
-        "tutor-discovery>=17.0.0,<18.0.0",
-        "tutor-mfe>=17.0.0,<18.0.0",
+        "tutor>=18.0.0,<19.0.0",
+        "tutor-discovery>=18.0.0,<19.0.0",
+        "tutor-mfe>=18.0.0,<19.0.0",
```

Comment:
should this not mention the PRs/branches here for now, as these versions won't be available until the actual release?

Reply:
Correction: This won't cause problem on sandbox as tutor redwood will be installed before installing this branch.